snippets/logging-stable-updates-snip.adoc

```diff
@@ -8,5 +8,5 @@
 
 [NOTE]
 ====
-The `stable` channel only provides updates to the most recent release of logging. To continue receiving updates for prior releases, you must change your subscription channel to `stable-X` where `X` is the version of logging you have installed.
+The `stable` channel only provides updates to the most recent release of an Operator. To continue receiving updates for prior releases, you must change your subscription channel to `stable-X` where `X` is the version of the Operator you have installed.
```

Comment:
I can see how it could be read that way. 'for the most recent release' instead of 'to' make it clearer?

Reply:
I don't know if just changing `to`->`for` is enough. Maybe something like this?

```suggestion
The `stable` channel provides updates for the most recent release of an Operator and will change to newer releases automatically. To continue receiving updates for prior releases, you must change your subscription channel to `stable-X` where `X` is the version of the Operator you have installed.
```

Reply:
Note it's `stable-X.Y`,  maybe a more explicit wording would be clearer:

```suggestion
Subscribing to the `stable` channel will _automatically_ upgrade the operator to the latest released GA version.
This includes automatic major and minor version upgrades, as well as applying the latest patches.

Subscribing to `stable-X.Y` will automatically apply the latest patches for version `X.Y`  (example 5.2.1 -> 5.2.2).
It will _not_ automatically upgrade to a new major or minor release.
When you are ready to upgrade, change your subscription, for example from `stable-5.3` -> `stable-5.4`
 ```